docs/api-testing/use-cases/working-with-headers.md

```diff
@@ -0,0 +1,129 @@
+---
+id: working-with-headers
+title: Working With the Header (statusCode, metrics)
+sidebar_label: Working with the Headers
+description: How to use the response header in your tests
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+An HTTP response is made of a payload but also contains contextual information. Using API Testing you can also deal with the whole response envelope.
+When you're making an HTTP request in the composer, you're providing a variable name. The variable will host the entire response payload. When the operation completes, another variable called `<variable_name>_response` is also created. Therefore various pieces of information such as HTTP header and metrics are contained in that variable.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+## Retrieving the statusCode
+
+Let's say `payload` is the name of the variable that contains the entire response payload.
+
+So by referencing the `payload_response.statusCode` expression you can access the status code. That's can be useful not only to check that a specific status code has been returned in your test but also to perform your tests for different status codes.
```

Comment:
I am not sure if run is the right work, but perform did not sound right either. 
```suggestion
By referencing the `payload_response.statusCode` expression you can access the status code. This can be useful not only to check that a specific status code has been returned in your test but also to run your tests for different status codes.
```

Reply:
```suggestion
So by referencing the `payload_response.statusCode` expression you can access the status code. That's can be useful not only to check that a specific status code has been returned in your test but also to execute your tests for different status codes.
```